lib/trento/operations/host_policy.ex

```diff
@@ -50,6 +54,58 @@ defmodule Trento.Operations.HostPolicy do
     |> OperationsHelper.reduce_operation_authorizations(databases_authorized)
   end
 
+  # Based on the maintenance procedures for HANA clusters, it's clear to me the following:
```

Comment:
Hey @balanza @abravosuse 
Watching this, I see that we completely ignore the running status of the SAP workloads and only consider the cluster status. So we only have a partial implementation of the policy. 
This part of the acceptance criteria is missing:
```
If there is an SAP workload in the host, it is stopped (HANA instance, SAP instance)
```

We can have:
1. A host without a cluster but a SAP workload running, so we should forbid the operation
2. A host belonging a cluster, the cluster stopped, and maybe for some reason the SAP workload is running (corner case, but It can happen, once the cluster is stopped, nothing forbids the user to start SAP instances).

So, we need to add this condition to the policy